src/webapp/js/feedback.js

```diff
@@ -110,7 +110,13 @@
             });
         } else if (CONTENT === state) {
 
-            feedback.utils.renderTemplate(state, { siteExists: feedback.siteExists, siteId: feedback.siteId, siteUpdaters: feedback.siteUpdaters, loggedIn: loggedIn, technicalToAddress: feedback.technicalToAddress, contactName: feedback.contactName}, 'feedback-content');
+            var plugins = '';
+            for(var i = 0; i<navigator.plugins.length; i++) {
+                plugins += navigator.plugins[i].name + ", ";
+            }
+
+            feedback.utils.renderTemplate(state, { plugins : plugins, screenWidth: screen.width, screenHeight: screen.height, oscpu: navigator.oscpu, windowWidth: window.outerWidth,
+                windowHeight: window.outerHeight, siteExists: feedback.siteExists, siteId: feedback.siteId, siteUpdaters: feedback.siteUpdaters, loggedIn: loggedIn, technicalToAddress: feedback.technicalToAddress, contactName: feedback.contactName}, 'feedback-content');
```

Comment:
That was already there.  We only need 'url' for the TECHNICAL state because we want to log clicking on the 2nd and 3rd boxes in the db as different things.